libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Utilities/CSFInternalDefines.m

```diff
@@ -32,6 +32,13 @@
 @import LaunchServices;
 #endif
 
+NSInteger kCSFNetworkLogContext = 0;
```

Comment:
Note that this is the recommended way to initialize a logging context from an identifier.  This way it gets initialized at load-time, but isn't bound to any particular class.